### DISCUSSION TITLE
The defination of NFT and Inscriptions :are inscriptions NFTs?

### DISCUSSION THREAD

**Discussion by DrJingLee** [ordinals/ord#2399] [Q&A]
> **[Why are sat inscriptions called "digital artifacts" instead of "NFTs"?](https://docs.ordinals.com/faq.html#why-are-sat-inscriptions-called-digital-artifacts-instead-of-nfts)**
> 
> **An inscription is an NFT**, but the term "digital artifact" is used instead, because it's simple, suggestive, and familiar.
> 
> The phrase "digital artifact" is highly suggestive, even to someone who has never heard the term before. In comparison, NFT is an acronym, and doesn't provide any indication of what it means if you haven't heard the term before.
> 
> Additionally, "NFT" feels like financial terminology, and the both word "fungible" and sense of the word "token" as used in "NFT" is uncommon outside of financial contexts.

The handbook and casey's video say Inscriptions are NFTs but list a lot differences with NFTs. And We all know Casey express his dislike with "NFTs" publicly. The handbook "FAQ" section also compare the "Inscriptions" with ETH "NFTs" and **gave the readers a strong impressions** that "Inscriptions"and "ETH NFTs"has a lot of **fundamental** differences.

This could be very much confusing to the new people who are coming into Ordinals. We try to educate people to understand what is the different between "Digital artifacts" or "Inscriptions" and "NFTs". While some service providers also have different understanding , for instance unisat using "inscriptions = brc20 + collections ", other products might design the nevigations bar with "inscriptions= NFTs + BRC20 . "

**Are inscriptions NFTs?
Are BRC20s NFTs?**

To make it clear ,We should let people understand clearly about the different definations .

Here are some of formulas of definitions :

1,NFTs > Inscriptions  
2,Inscriptions = digital artifacts ," **inscriptions is digital arttifacts "**
3,digital artifacts > Inscriptions;

From 1-3, We know 
4, NFT > Digital artifacts >= Inscriptions  

We can also learn from the handbook says :**"What are digital artifacts? Simply put, they are the digital equivalent of physical artifacts."**
From the definition we know digital artificial are the digital equivalent of physcial artficats, they not necessary be on Chain,most of them are just stored in a offline. So only Onchain Digital artifacts are the subset of NFT.

5.1,digital artifacts > NFTs           
Or 
5.2 On Chain Digital Artifacts < NFT

And With the development of BRC20 ,it making even more confusing :

6,Inscriptions = BRC 20 + other "digital artifacts " (Such as documents ,audio vedio metaverse etc) or
We put 

7,Inscriptions = BRC20s(Non ART) + BTC NFTs (ART)

Given 

8 Inscriptions > BRC20 

and

9, NFT>Inscriptions

So We can conclude that 

10, NFT > Inscriptions > BRC 20 ,

and conclude BRC20s are NFTs which go against common sense.

BRC20 is an NFT or FT ?

@casey Thought ?

**Please post your comments below**

**Comment by casey** [comment]
Inscriptions are definitely NFTs. Digital artifacts are a subset of NFT, so inscriptions are both.

**Comment by DrJingLee** [comment]
> Inscriptions are definitely NFTs. Digital artifacts are a subset of NFT, so inscriptions are both.
**If its correct , the relationship will looks like this :**
1,
<img width="600" alt="图片" src="https://github.com/ordinals/ord/assets/95461562/9ea064b6-1d20-414e-a12f-bbcf072f22ce">

2,
**But  given:**
<img width="600" alt="图片" src="https://github.com/ordinals/ord/assets/95461562/66209d24-6cd9-4d4b-83c5-f823390a9486">

**This will lead a conclusion of BRC20s are NFTs as well.**

3,And this chart is refering the **broader defination of digital artifacts defined by the handbook** 

<img width="711" alt="图片" src="https://github.com/ordinals/ord/assets/95461562/8672a170-2c8b-4758-8c45-572876d34ad3">

4,Personally I think this chart is more likely to describe the current eco development:

<img width="711" alt="图片" src="https://github.com/ordinals/ord/assets/95461562/f9eefd09-f3e9-4ede-9271-545630cfe45f">

**Comment by DrJingLee** [comment]
![图片](https://github.com/ordinals/ord/assets/95461562/67e9b665-6afd-42ab-ad33-1ce8bdacb94d)

I did a pull on my twitter and seems 50% of them agree the BRC20 ORC and ORC Cash are NFTs, and the 50% people dont agree or dont know....
